python/mxnet/_ctypes/ndarray.py

```diff
@@ -31,10 +31,10 @@
 
 class NDArrayBase(object):
     """Base data structure for ndarray"""
-    __slots__ = ["handle", "writable"]
+    __slots__ = ["handle", "writable", "dlpack_handle"]
```

Comment:
do not put dlpack_handle inside it, instead directly return a pycapsule that contains the handle